### ISSUE TITLE
Excalidraw causes entire page to flicker on render.

### ISSUE THREAD

**Issue by wuguishifu** [excalidraw/excalidraw#8636]
When adding an Excalidraw component to anywhere on a React page, it causes the entire page to flicker. For example, in the [Excalidraw API documentation](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/installation), if you toggle between the "installation" and "integration" pages, you can see that when you click on the "integration" page the entire page flickers. This happens for every page that contains an Excalidraw canvas, and does not happen for the pages that do not.

Has anyone else noticed this and figured out a way to mitigate it? Is this only on my end? I have tested this via my own project and the Excalidraw API docs on Brave, Chrome, Edge, and Firefox, both on Mac and Windows, and  it happens on every browser.

**Comment by wuguishifu** [comment]
For additional information, it looks like this is caused by the "loading scene..." screen which looks like it's taking up 100% of the screen, so this may be an issue of bad CSS. I'm going to investigate more and if I can find a hotfix I'll make a PR.

**Comment by wuguishifu** [comment]
I don't have a ton of time to make a PR right now, but here's what I found out. It looks like this is being caused by [this component](https://github.com/excalidraw/excalidraw/blob/47ee8a00945793340bb20715b2f383a4c3da2139/packages/excalidraw/components/LoadingMessage.tsx#L8) and [this css](https://github.com/excalidraw/excalidraw/blob/47ee8a00945793340bb20715b2f383a4c3da2139/packages/excalidraw/css/app.scss#L14). This is causing a grey rectangle to cover the entire screen while it's loading. Frankly, I don't think this really makes sense from a UI perspective, especially if the `<Excalidraw />` component doesn't take up the full screen. I also think that this wasn't an intentional feature, as it looks like Excalidraw has a separate "loading scene..." display that's embedded in the actual `<Excalidraw />` element. I was able to just kill this in my app using this css in my index.css file:

```css
.LoadingMessage,
.LoadingMessage--dark {
  display: none !important;
}
```
